diff --git a/toolkit/locales/en-US/toolkit/branding/brandings.ftl b/toolkit/locales/en-US/toolkit/branding/brandings.ftl
index d390fc6e8e..6b9b96ae5e 100644
--- a/toolkit/locales/en-US/toolkit/branding/brandings.ftl
+++ b/toolkit/locales/en-US/toolkit/branding/brandings.ftl
@@ -38,20 +38,20 @@
 # The particle "by" can be localized, "Fakespot" and "Mozilla" should not be localized or transliterated.
 -fakespot-brand-full-name = Fakespot by Mozilla
 
-# “Suggest” can be localized, “Firefox” must be treated as a brand
+# “Suggest” can be localized, “Bento” must be treated as a brand
 # and kept in English.
--firefox-suggest-brand-name = Firefox Suggest
+-firefox-suggest-brand-name = Bento Suggest
 
-# ”Home" can be localized, “Firefox” must be treated as a brand
+# ”Home" can be localized, “Bento” must be treated as a brand
 # and kept in English.
--firefox-home-brand-name = Firefox Home
+-firefox-home-brand-name = Bento Home
 
-# View" can be localized, “Firefox” must be treated as a brand
+# View" can be localized, “Bento” must be treated as a brand
 # and kept in English.
--firefoxview-brand-name = Firefox View
+-firefoxview-brand-name = Bento View
 
-# Firefox Labs is the name for a page in Settings to allow users to learn about
+# Bento Labs is the name for a page in Settings to allow users to learn about
 # experimental and in-development features, and turn those features on and off.
-# The "Labs" portion can be localized, “Firefox” must be treated as a brand
+# The "Labs" portion can be localized, “Bento” must be treated as a brand
 # and kept in English.
--firefoxlabs-brand-name = Firefox Labs
+-firefoxlabs-brand-name = Bento Labs
