From ece5076a1ed1c8007a94a464562caa661ca34fde Mon Sep 17 00:00:00 2001
From: Bento Patch Stack <patch-stack@bentobrowser.invalid>
Date: Wed, 24 Jun 2026 19:23:28 +1000
Subject: [PATCH] Register Bento key actors

---
 browser/actors/moz.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/browser/actors/moz.build b/browser/actors/moz.build
index 74ce9247c8a85b1ef71568c8055d9992b757e573..028421f9773113cb0aef8f6336d1a49454a2359f 100644
--- a/browser/actors/moz.build
+++ b/browser/actors/moz.build
@@ -103,6 +103,8 @@ FINAL_TARGET_FILES.actors += [
     "AboutReaderParent.sys.mjs",
     "AboutTabCrashedChild.sys.mjs",
     "AboutTabCrashedParent.sys.mjs",
+    "BentoKeyChild.sys.mjs",
+    "BentoKeyParent.sys.mjs",
     "BlockedSiteChild.sys.mjs",
     "BlockedSiteParent.sys.mjs",
     "BrowserProcessChild.sys.mjs",
-- 
2.50.1 (Apple Git-155)

